docs/examples/hello_pt_tb.rst

```diff
@@ -0,0 +1,126 @@
+Quickstart (PyTorch with TensorBoard)
+=====================================
+
+Introduction
+-------------
+
+In this exercise, you will learn how to stream TensorBoard events from the clients to the server in order to visualize live training metrics from a central place on the server.
+This exercise will be working with the ``hello-pt-tb`` application in the examples folder, which builds upon :doc:`hello_pt` by adding TensorBoard streaming.
+
+.. note::
+
+  This exercise also differs from :doc:`hello_pt`, as it uses the ``Learner`` API along with the ``LearnerExecutor``.
+  In short, the execution flow is abstracted away into the ``LearnerExecutor``, allowing you to only need to implement the required methods in the ``Learner`` class.
+  This will not be the focus of this guide, however you can learn more at :class:`Learner<nvflare.app_common.abstract.learner_spec.Learner>`
+  and :class:`LearnerExecutor<nvflare.app_common.executors.learner_executor.LearnerExecutor>`.
+
+
+Let's get started. Make sure you have an environment with NVIDIA FLARE installed as described in
+:doc:`installation <../installation>` guide. First clone the repo:
+
+.. code-block:: shell
+
+  $ git clone https://github.com/NVIDIA/NVFlare.git
+
+Now remember to activate your NVIDIA FLARE Python virtual environment from the installation guide.
+Since you will use PyTorch, torchvision, and TensorBoard for this exercise, let's go ahead and install these libraries:
+
+.. code-block:: shell
+
+  (nvflare-env) $ python3 -m pip install torch torchvision tensorboard
+
+
+Adding TensorBoard Streaming to Configurations
+------------------------------------------------
+
+Inside the config folder there are two files, ``config_fed_client.json`` and ``config_fed_server.json``.
+
+.. literalinclude:: ../../examples/hello-pt-tb/config/config_fed_client.json
+   :language: json
+   :linenos:
+   :caption: config_fed_client.json
+
+Take a look at the components section at line 24. The first component is the ``pt_learner`` which contains the initialization, training, and validation logic.
+``pt_learner.py`` is where we will add our TensorBoard streaming changes.
+
+Next we have the :class:`AnalyticsSender<nvflare.app_common.widgets.streaming.AnalyticsSender>`, which implements some common methods that follow the signatures from the PyTorch SummaryWriter.
+This makes it easy for the ``pt_learner`` to log metrics and send events.
+
+Finally, we have the :class:`ConvertToFedEvent<nvflare.app_common.widgets.convert_to_fed_event.ConvertToFedEvent>`, which converts local events to federated events.
+This changes the ``analytix_log_stats`` events to be federated events, which can then be streamed from the clients to the server.
```

Comment:
Suggest to mention the event is converted into ``fed.analytix_log_stats`` here as you did in the README file.

Reply:
Done

Reply:
Done